### ISSUE TITLE
add_hparam creates subdirectory

### ISSUE THREAD

**Issue by koelscha** [lanpa/tensorboardX#519]
add_hparam creates a subdirectory with name time.time(). In tensorboard, this shows up as an individual run. Is this intended behavior? I think it would be much cleaner to put it in the same events file, or -if that is not possible- at least in the same directory of the actual run.

**Comment by versatran01** [comment]
Yeah, I was also wondering why add_hparam creates a subdir for every call.
It's not like I will change any hyper-parameters (like batch size) during each run.
Would like to hear the reason behind this.

**Comment by lanpa** [comment]
https://github.com/pytorch/pytorch/issues/32651#issuecomment-579621209
